docs/framework/data/adonet/ef/toc.yml

```diff
@@ -5,110 +5,89 @@
 - name: Modeling and Mapping
   href: modeling-and-mapping.md
   items:
-  - name: EDM Generator (EdmGen.exe)
-    href: edm-generator-edmgen-exe.md
-    items:
-    - name: "How to: Use EdmGen.exe to Generate the Model and Mapping Files"
-      href: how-to-use-edmgen-exe-to-generate-the-model-and-mapping-files.md
-    - name: "How to: Use EdmGen.exe to Generate Object-Layer Code"
-      href: how-to-use-edmgen-exe-to-generate-object-layer-code.md
-    - name: "How to: Use EdmGen.exe to Validate Model and Mapping Files"
-      href: how-to-use-edmgen-exe-to-validate-model-and-mapping-files.md
-  - name: "How to: Define the Connection String"
-    href: how-to-define-the-connection-string.md
-  - name: "How to: Make Model and Mapping Files Embedded Resources"
-    href: how-to-make-model-and-mapping-files-embedded-resources.md
-  - name: Working with Data Definition Language
-    href: working-with-data-definition-language.md
+    - name: EDM Generator (EdmGen.exe)
+      href: edm-generator-edmgen-exe.md
+      items:
+        - name: "How to: Use EdmGen.exe to Generate the Model and Mapping Files"
+          href: how-to-use-edmgen-exe-to-generate-the-model-and-mapping-files.md
+        - name: "How to: Use EdmGen.exe to Generate Object-Layer Code"
+          href: how-to-use-edmgen-exe-to-generate-object-layer-code.md
+        - name: "How to: Use EdmGen.exe to Validate Model and Mapping Files"
```

Comment:
This indentation is peculiar. The rest of the files in this repo (and others that I've seen) do not indent unnecessarily. I would fix all the indentation in this file to be more inline with the other toc files. While it feels unnatural to NOT indent, you don't need to indent child items. This is perfectly acceptable (and seems to be the preferred way of indentation) where the `-` character of a new items is aligned to the left-column of the parent.

```yml
- name: Forms
  items:
  - name: Overview
    href: file.md
  - name: Event handlers
    href: forms/events.md
  - name: Automatic scaling
    href: forms/autoscale.md
  - name: Common tasks
    items:
    - name: Add a form
      href: forms/how-to-add.md
```

Reply:
@adegeo @tdykstra is there a standard formatter? without the indentation, it wasn't rendering the TOC correctly. I just used VS Code "format document" then it rendered fine. What's the recommended way to ensure the file formats correctly without manually indenting each line?

Reply:
My guess as to why it didn't render correctly was just that the system probably inferred spacing based on the other entries, so you can't mix/match indentation, you have to go with one unit of measurement. These are also just my observations 😄 probably any indentation works as long as the file is uniform.

I just like consistency, especially throughout the same repo. Here is the FSharp file for example: https://github.com/dotnet/docs/blob/master/docs/fsharp/toc.yml

Reply:
> What's the recommended way to ensure the file formats correctly without manually indenting each line?

Manually indenting is the only way I know to get indentation consistent with the other toc.yml files in the docs repo.

Reply:
Should be good.